elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -10,30 +10,23 @@ $(function() {
 });
 </script>
 
-    <div class="row">
-        <div class="column {{dropzone_layout}}">
-            <ul class="list-group-item list-group-item-primary header">
-                <li><strong>{{source-header}}</strong>
-                    <a role="button" class="btn btn-link btn-sm disabled" data-toggle="popover">
-                        <i class="fa fa-question-circle" aria-hidden="true" style="visibility: hidden;"></i>
-                    </a>
-                </li>
-            </ul>
-            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}">
+    <div style="display:flex;flex-wrap:wrap;">
```

Comment:
Nit: we generally err towards using Bootstrap classes instead of inline styles. e.g.:

```suggestion
    <div class="d-flex flex-wrap">
```

It doesn't make much difference in practice, but it's a convention nonetheless 🙂